### DISCUSSION TITLE
Having issues using this on Nextjs

### DISCUSSION THREAD

**Discussion by dexter4life** [project-nv/night-vision#60] [General]
Hello, i appreciate your effort is really nice. I have tried everything to use this on Nextjs project keep getting this issue 
`windows` is not defined. What can I do to get rid of this?

**Comment by dexter4life** [comment]
![image](https://user-images.githubusercontent.com/10168414/219363137-96e7f875-37c1-4c4d-8652-effcd596d7e4.png)

**Comment by oneart-dev** [comment]
I assume you have to be sure not to run this lib in SSR mode as window is undefined in node environment on server.
